### DISCUSSION TITLE
Error in syntax

### DISCUSSION THREAD

**Discussion by aqilamaulidiyah** [Jingkang50/OpenOOD#124] [General]

**Comment by Jingkang50** [comment]
@Prophet-C Pengyun, let's correct it.

**Comment by aqilamaulidiyah** [comment]
https://github.com/Jingkang50/OpenOOD/blob/74dd3d4b258a306465c682da6cccba01c607da83/openood/postprocessors/vim_postprocessor.py#L65
Correction :
print(f'alpha = {self.alpha:.4f}')

Sorry I deleted it by accident
